src/main/java/core/basesyntax/FigureTypeSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureTypeSupplier {
+    private Random random = new Random();
+
+    public FigureType getFigureType() {
```

Comment:
this method actually provides random figure type, so please rename is correspondingly